frontend/src/scenes/retention/RetentionGraph.tsx

```diff
@@ -61,7 +61,6 @@ export function RetentionGraph({ inSharedMode = false }: RetentionGraphProps): J
             isStacked={retentionFilter?.display !== ChartDisplayType.ActionsBar}
             trendsFilter={{ aggregationAxisFormat: 'percentage' } as TrendsFilter}
             tooltip={{
-                rowCutoff: 11, // 11 time units is hardcoded into retention insights
```

Comment:
so what will happen without this?

Reply:
sorry should have put a screenshot for this.
This fixes Q2 in the zendesk ticket. Regarding tooltip now showing up for values 12 and later